roles/lnls-ans-role-servweb/tasks/main.yml

```diff
@@ -0,0 +1,7 @@
+---
+- name: Clone and Install Control System Constants repository
+  include_tasks: clone-repo.yml
+  with_items:
+    - "{{ sirius_repos }}"
+  when:
+    - sirius_repos is defined
```

Comment:
Same as above. I think it should be better to use the convention of defining variables with the role name as prefix . In this way, we avoid having to deal with a variable being redefined by another role. So, sirius_repos, should be servweb_repos

Reply:
deleted